src/core/cubic.c

```diff
@@ -426,6 +507,52 @@ CubicCongestionControlOnDataAcknowledged(
         }
     }
 
+    if (AckEvent->MinRttValid) {
+        uint32_t Eta;
+
+        if (Cubic->HyStartState != HYSTART_DONE) {
+            //
+            // Update Min RTT for the first N ACKs.
+            //
+            if (Cubic->HyStartAckCount < CubicHyStartNSampling) {
+                Cubic->MinRttInCurrentRound =
+                    CXPLAT_MIN(
+                        Cubic->MinRttInCurrentRound,
+                        AckEvent->MinRtt);
+                Cubic->HyStartAckCount++;
+            } else {
+                Eta =
+                    CXPLAT_MIN(CubicHyStartMaxEta,
+                        CXPLAT_MAX(CubicHyStartMinEta, Cubic->MinRttInLastRound / 8));
+
+                if (Cubic->HyStartState == HYSTART_NOT_FOUND) {
+                    //
+                    // Looking for delay increase.
+                    //
+                    if (Cubic->MinRttInLastRound != UINT32_MAX &&
+                        Cubic->MinRttInCurrentRound != UINT32_MAX &&
+                        (Cubic->MinRttInCurrentRound >= Cubic->MinRttInLastRound + Eta)) {
+                        //
+                        // We found an exit. Now we are going to do Conservative Slow Start for
+                        // ConservativeSlowStartRounds rounds.
+                        //
+                        CubicCongestionHyStartChangeState(Cc, HYSTART_FOUND);
+                        Cubic->CWndSlowStartGrowthDivisor = ConservativeSlowStartGrowthDivisor;
```

Comment:
>CWndSlowStartGrowthDivisor [](http://example.com/codeflow?start=31&length=26)

It doesn't look like this variable has any effect. The cwnd growth logic doesn't use it.